CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Next Version
+
+# Version 4.8.0
 - Add support of Android 12L (API level 32)
 - Bump AdMob to v21.2.0
+- Bump minSdkVersion to 19
```

Comment:
By doing that you can remove all the degraded mode of the SDK:
- https://github.com/criteo/android-publisher-sdk/blob/main/publisher-sdk/src/main/java/com/criteo/publisher/util/DeviceUtil.java#L65 become true and could be inlined everywhere, then removed
- https://github.com/criteo/android-publisher-sdk/blob/main/publisher-sdk/src/main/java/com/criteo/publisher/DummyCriteo.java could be removed
- CriteoInternal class could be inlined in the Criteo abstract class
- Many tests about degraded features could be removed

Reply:
Let's keep the degraded mode to simplify API deprecation in future

Reply:
LGTM

Reply:
Done